### DISCUSSION TITLE
Install teleport behind Kong proxy

### DISCUSSION THREAD

**Discussion by PallaviPalisetti** [gravitational/teleport#10748] [Ideas]
1. we have kubernetes infrastructure applications that are configured with the kong-proxy(kong ingress controller) & here teleport acts as a proxy for infrastructure applications, is there a way to configure teleport as a ClusterIP proxied by kong ingress controller.

2. Because by default teleport service opens multiple ports & we can't configure same path under ingress controller for different ports.

**Comment by webvictim** [comment]
There's not currently any support in the `teleport-cluster` Helm chart for using an Ingress. You can change the type of `Service` which is configured here: https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-cluster/values.yaml#L213-L218

You could also consider forking the chart to add support for using an `Ingress`.
